catch_changes.patch

```diff
@@ -0,0 +1,33 @@
+Index: src/main/cpp/bls-signatures/depends/catch2/include/catch2/catch.hpp
+IDEA additional info:
+Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
+<+>UTF-8
```

Comment:
there are two compile time errors in the `bls-signature` library.  Part of it is due to the version of `gclib`.  Instead of forking this repo, for now we can simply patch the files that need to be changed.